public/css/main.css

```diff
@@ -1,20 +1,357 @@
 @font-face {
   font-family: 'squealerregular';
   src: url('fonts/squealer-webfont.eot');
-  src: url('fonts/squealer-webfont.eot?#iefix') format('embedded-opentype'),
-    url('fonts/squealer-webfont.woff2') format('woff2'),
-    url('fonts/squealer-webfont.woff') format('woff'),
-    url('fonts/squealer-webfont.ttf') format('truetype'),
-    url('fonts/squealer-webfont.svg#squealerregular') format('svg');
+  src: url('fonts/squealer-webfont.eot?#iefix') format('embedded-opentype'), url('fonts/squealer-webfont.woff2') format('woff2'), url('fonts/squealer-webfont.woff') format('woff'), url('fonts/squealer-webfont.ttf') format('truetype'), url('fonts/squealer-webfont.svg#squealerregular') format('svg');
   font-weight: normal;
   font-style: normal;
 }
 
 body {
   background-image: radial-gradient(circle, #3241a6 0, #202a6b 120%);
-  font-family: "proxima-nova", -apple-system, BlinkMacSystemFont, "Segoe UI",
-    "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", "Helvetica Neue",
-    "Arial", sans-serif;
+  font-family: "proxima-nova", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", "Helvetica Neue", "Arial", sans-serif;
+  padding: 0;
+  margin: 0;
```

Comment:
Can we remove the padding/margin here and declare on the body tag using tachyons classes? http://tachyons.io/docs/